client/docs/modules.md

```diff
@@ -0,0 +1,39 @@
+# Monorepo: Modules
+
+Modules are the packages that can be found in the `modules/` folder. They're intended for pure ESM output, and are compiled rapidly using Estrella, an esbuild-based utility. They have an important file structure (don't change it), which looks like the following:
+
+```
+module:
+- src/
+  - index.ts
+  - **/*.ts
+- tests/
+  - *.ts
+- package.json
+- tsconfig.json
+```
+
+The `src/` folder is compiled into a `dist/` folder when the module is built. Similarly, files in the `tests/` folder are compiled into the `tests/dist/` folder when the tests are built.
```

Comment:
n/b: Is it `tests/dist/`, or does it go in `dist/tests/`?

Reply:
It goes into `tests/dist` because `dist/` is effectively the emitted content of the NPM package, which shouldn't have built test files in it.